lib/web/integrations.go

```diff
@@ -220,7 +221,7 @@ func (h *Handler) integrationStats(w http.ResponseWriter, r *http.Request, p htt
 		return nil, trace.Wrap(err)
 	}
 
-	summary, err := collectAWSOIDCAutoDiscoverStats(r.Context(), ig, clt.DiscoveryConfigClient())
+	summary, err := collectAWSOIDCAutoDiscoverStats(r.Context(), h.logger, ig, clt.DiscoveryConfigClient(), clt, clt.IntegrationAWSOIDCClient())
```

Comment:
I think it's time to either convert the list of parameters to a struct or make this function a member of some other object so we don't have to pass 6 parameters to it (e.g. 3 of those are obtained from `clt`).